.github/workflows/deploy_prod.yml

```diff
@@ -35,6 +38,8 @@ jobs:
     steps:
       - name: 'Checkout'
         uses: 'actions/checkout@v4'
+        with:
+          ref: release # specify release branch is checked out
```

Comment:
If not set, then it will deploy the branch you have checked out(feature/working branch). 

I did not set this for dev and qa for this purpose because we want to deploy the working branch